### DISCUSSION TITLE
update framework to 3.1.6

### DISCUSSION THREAD

**Discussion by vieuxcampus** [templaza/astroid-framework#828] [Q&A]
Hello, I use template one 2.6.4 . I made an update of the framework to 3.1.6. It causes a big mess and  it gives an Uncaught TypeError:
[in jquery](http://localhost/media/vendor/jquery/js/jquery.min.js?fb6b2a5a55e370ce0e4e8b69fd245f57)
it's related to bootstrap  tooltip
can somebody help ?
Thanks

**Comment by sonvnn** [comment]
Can you screenshot your issue here?

**Reply by vieuxcampus** [reply]
![Capture](https://github.com/user-attachments/assets/509259c5-5e92-479b-8064-b68d0d10f670)

**Reply by sonvnn** [reply]
@vieuxcampus Let's try to go to Bootstrap Settings > Select tooltip and try it again. https://docs.astroidframe.work/basic-settings/bootstrap-settings
